man/hwdb-usb-device.c

```diff
@@ -9,7 +9,7 @@ int print_usb_properties(uint16_t vid, uint16_t pid) {
   int r;
 
   /* Match this USB vendor and product ID combination */
-  snprintf(match, sizeof match, "usb:v%04Xp%04X", vid, pid);
+  xsprintf(match, "usb:v%04Xp%04X", vid, pid);
```

Comment:
No, this is in man, we don't expose xsprintf to users.

Reply:
Thanks, fixed. I also added a rule to skip the `xsprintf` trasnformation on `man/*`.